src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -116,7 +115,13 @@ class goto_instrument_parse_optionst:
   bool partial_inlining_done;
   bool remove_returns_done;
 
+  symbol_tablet symbol_table;
```

Comment:
why not `goto_modelt` as done in `cbmc_parse_options.h` ?

Reply:
Done, big job.